nni/common/concrete_trace_utils/concrete_proxy.py

```diff
@@ -58,6 +58,7 @@ class ConcreteProxy(Proxy):
     op_call_ex = dis.opmap['CALL_FUNCTION_EX']
     op_not = dis.opmap['UNARY_NOT']
     op_unpack_sequence = dis.opmap['UNPACK_SEQUENCE']
+    op_dict_merge = dis.opmap['DICT_MERGE']
```

Comment:
dont know why **kwargs call DICT_MERGE first before UNPACK_SEQUENCE...
but this hack should work for simple cases

Reply:
likewise, if i call `dict(**kwargs)``, python calls DICT_MERGE first also